README.md

```diff
@@ -265,7 +275,7 @@ You need to switch from GUI which is running X11 or Wayland to the Character Use
 
 ```Shell
 $ Ctrl + Alt + F3 # Switching to CUI
-$ sudo FLUTTER_DRM_DEVICE="/dev/dri/card1" ./flutter-drm-backend ./sample/build/linux/x64/release/bundle
+$ sudo FLUTTER_DRM_DEVICE="/dev/dri/card1" <./binary_file_name> ./sample/build/linux/x64/release/bundle
```

Comment:
<./binary_file_name> -> <binary_file_name>

Reply:
I fixed it.